components/technology/modules/ROOT/pages/differences.adoc

```diff
@@ -0,0 +1,103 @@
+= SKALE Chain Differences from Ethereum
+
+SKALE Chains operate like Ethereum, and there are a few other features and differences that make SKALE Chains even more powerful and suitable to run your dApp. 
+
+== Zero gas fee transactions
+
+Transactions on SKALE Chain run in without any gas fee. This means there is gas on SKALE Chains just like on Ethereum, but it's powered by what's called SKALE FUEL (sFUEL), which has no monetary value. In this way, executing transactions and state changes don't incur costs but require consuming gas in sFUEL, which is value-less. SKALE Chain gas provides a way to meter or limit transactions on the SKALE Chain to prevent malicious execution.
+
+== Empty Block Intervals
+
+Your SKALE Chain will continue to mine blocks in the absence of transactions. Empty blocks are used to signal to the SKALE Network that your SKALE Chain is healthy and operational. The default empty block interval is 10 seconds. When transactions fill blocks, blocks are mined as fast as possible, up to sub-second blocks depending on validator-node performance.
+
+== Block Gas Limit
+
+SKALE Chains have a block gas limit of 268435455.
+
+== Contract Code Size Limit
+
+SKALE chains have a default code size limit of 64 kB, and are configurable at chain deployment to code size limits of 16, 32, 64, 128, 256, 512, 1024 kB and unlimited. (Ethereum has a contract code size limit of 24576 bytes.)
+
+== JSON-RPC Methods
+
+SKALE Chains have a modified JSON-RPC interface compared to Geth Ethereum clients. In general, this shouldn't raise any incompatibilities unless your dApp requires particular methods. 
+
+xref:json-rpc-interface.adoc[See the JSON-RPC interface Compatibility table for more information].
+
+== SKALE Chain Rotation
+
+SKALE Chains run across 16 randomly chosen nodes. Your SKALE Chain can be configured to randomly swap out nodes for other nodes in the network. This helps reduce any potential collusion over time. To the SKALE Chain owner and end-users, this is a process that runs behind the scenes...  The incoming node will gather all previous blocks, transactions, etc from the SKALE Chain in a catchup procedure, and the updated node set will form a new BLS public key through DKG, and all subsequent blocks will be signed with the new BLS public key.
```

Comment:
Suggestion, remove the three ..., replace with .

Reply:
accepted